Cargo.toml

```diff
@@ -54,6 +54,18 @@ panic = "abort"
 codegen-units = 1
 lto = "fat"
 
+[patch.crates-io]
+alloy-eips = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-serde = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-provider = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-consensus = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-transport = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-rpc-types = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-rpc-client = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-node-bindings = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-transport-http = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
+alloy-rpc-types-engine = { git = "https://github.com/refcell/alloy", branch = "rf/feat/indexed-blob-hash" }
```

Comment:
TODO: Remove these and bump alloy dep versions once a release is published that includes the `IndexedBlobHash` type.